{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<h3 id=\"summary\">Summary</h3>\n<p>Found an issue: Call to requests with verify=False disabling SSL certificate checks, security issue. Make the impact and severity as straightforward as possible. This rule enforces always verifying SSL certificates for methods in the Requests library. Certificates are validated by default which is the desired behavior.</p>\n<p>Encryption in general is typically critical to the security of many applications. Using TLS can significantly increase security by guaranteeing the identity of the party you are communicating with. This is accomplished by one or both parties presenting trusted certificates during the connection initialization phase of TLS.</p>\n<p>It is important to note that modules such as httplib within the Python standard library did not verify certificate chains until it was fixed in 2.7.9 release.</p>\n<h3 id=\"details\">Details</h3>\n<p>Severity: Critical</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/mindsdb/mindsdb/security/advisories/GHSA-8hx6-qv6f-xgcw\">https://github.com/mindsdb/mindsdb/security/advisories/GHSA-8hx6-qv6f-xgcw</a></li>\n<li><a href=\"https://github.com/mindsdb/mindsdb/commit/083afcf6567cf51aa7d89ea892fd97689919053b\">https://github.com/mindsdb/mindsdb/commit/083afcf6567cf51aa7d89ea892fd97689919053b</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-8hx6-qv6f-xgcw\">https://github.com/advisories/GHSA-8hx6-qv6f-xgcw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-8hx6-qv6f-xgcw",
            "title": "[MindsDB] MindsDB 'Call to requests with verify=False disabling SSL certificate checks, security issue.' issue",
            "date_modified": "2023-08-01T19:55:15.000Z",
            "date_published": "2023-08-01T19:55:14.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-8hx6-qv6f-xgcw"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OS Command Injection in GitHub repository mlflow/mlflow prior to 2.6.0.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2023-4033\">https://nvd.nist.gov/vuln/detail/CVE-2023-4033</a></li>\n<li><a href=\"https://github.com/mlflow/mlflow/commit/6dde93758d42455cb90ef324407919ed67668b9b\">https://github.com/mlflow/mlflow/commit/6dde93758d42455cb90ef324407919ed67668b9b</a></li>\n<li><a href=\"https://huntr.dev/bounties/5312d6f8-67a5-4607-bd47-5e19966fa321\">https://huntr.dev/bounties/5312d6f8-67a5-4607-bd47-5e19966fa321</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-ffw3-6378-cqgp\">https://github.com/advisories/GHSA-ffw3-6378-cqgp</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-ffw3-6378-cqgp",
            "title": "[mlflow] mlflow vulnerable to OS Command Injection",
            "date_modified": "2023-08-01T19:44:28.000Z",
            "date_published": "2023-08-01T03:31:17.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-ffw3-6378-cqgp"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Identity (Keystone) before 2014.1.5 and 2014.2.x before 2014.2.4 logs the backend_argument configuration option content, which allows remote authenticated users to obtain passwords and other sensitive backend information by reading the Keystone logs.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-3646\">https://nvd.nist.gov/vuln/detail/CVE-2015-3646</a></li>\n<li><a href=\"https://bugs.launchpad.net/keystone/+bug/1443598\">https://bugs.launchpad.net/keystone/+bug/1443598</a></li>\n<li><a href=\"http://lists.openstack.org/pipermail/openstack-announce/2015-May/000356.html\">http://lists.openstack.org/pipermail/openstack-announce/2015-May/000356.html</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinapr2015-2511959.html\">http://www.oracle.com/technetwork/topics/security/bulletinapr2015-2511959.html</a></li>\n<li><a href=\"https://web.archive.org/web/20210122154200/http://www.securityfocus.com/bid/74456\">https://web.archive.org/web/20210122154200/http://www.securityfocus.com/bid/74456</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jwpw-ppj5-7h4w\">https://github.com/advisories/GHSA-jwpw-ppj5-7h4w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jwpw-ppj5-7h4w",
            "title": "[keystone] OpenStack Keystone Logs Passwords",
            "date_modified": "2023-08-01T00:13:51.000Z",
            "date_published": "2022-05-13T01:26:10.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jwpw-ppj5-7h4w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Identity (Keystone) before 2014.1.5 and 2014.2.x before 2014.2.4 logs the backend_argument configuration option content, which allows remote authenticated users to obtain passwords and other sensitive backend information by reading the Keystone logs.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-3646\">https://nvd.nist.gov/vuln/detail/CVE-2015-3646</a></li>\n<li><a href=\"https://bugs.launchpad.net/keystone/+bug/1443598\">https://bugs.launchpad.net/keystone/+bug/1443598</a></li>\n<li><a href=\"http://lists.openstack.org/pipermail/openstack-announce/2015-May/000356.html\">http://lists.openstack.org/pipermail/openstack-announce/2015-May/000356.html</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinapr2015-2511959.html\">http://www.oracle.com/technetwork/topics/security/bulletinapr2015-2511959.html</a></li>\n<li><a href=\"https://web.archive.org/web/20210122154200/http://www.securityfocus.com/bid/74456\">https://web.archive.org/web/20210122154200/http://www.securityfocus.com/bid/74456</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jwpw-ppj5-7h4w\">https://github.com/advisories/GHSA-jwpw-ppj5-7h4w</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jwpw-ppj5-7h4w",
            "title": "[keystone] OpenStack Keystone Logs Passwords",
            "date_modified": "2023-08-01T00:13:51.000Z",
            "date_published": "2022-05-13T01:26:10.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jwpw-ppj5-7h4w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Murano before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), Murano-dashboard before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), and python-muranoclient before 0.7.3 (liberty) and 0.8.x before 0.8.5 (mitaka) improperly use loaders inherited from yaml.Loader when parsing MuranoPL and UI files, which allows remote attackers to create arbitrary Python objects and execute arbitrary code via crafted extended YAML tags in UI definitions in packages.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-4972\">https://nvd.nist.gov/vuln/detail/CVE-2016-4972</a></li>\n<li><a href=\"https://bugs.launchpad.net/murano/+bug/1586079\">https://bugs.launchpad.net/murano/+bug/1586079</a></li>\n<li><a href=\"https://bugs.launchpad.net/python-muranoclient/+bug/1586078\">https://bugs.launchpad.net/python-muranoclient/+bug/1586078</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2016/06/23/8\">http://www.openwall.com/lists/oss-security/2016/06/23/8</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-87r7-q54j-f9qg\">https://github.com/advisories/GHSA-87r7-q54j-f9qg</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg",
            "title": "[python-muranoclient] OpenStack Murano Code Execution",
            "date_modified": "2023-07-31T23:11:20.000Z",
            "date_published": "2022-05-17T03:48:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Murano before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), Murano-dashboard before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), and python-muranoclient before 0.7.3 (liberty) and 0.8.x before 0.8.5 (mitaka) improperly use loaders inherited from yaml.Loader when parsing MuranoPL and UI files, which allows remote attackers to create arbitrary Python objects and execute arbitrary code via crafted extended YAML tags in UI definitions in packages.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-4972\">https://nvd.nist.gov/vuln/detail/CVE-2016-4972</a></li>\n<li><a href=\"https://bugs.launchpad.net/murano/+bug/1586079\">https://bugs.launchpad.net/murano/+bug/1586079</a></li>\n<li><a href=\"https://bugs.launchpad.net/python-muranoclient/+bug/1586078\">https://bugs.launchpad.net/python-muranoclient/+bug/1586078</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2016/06/23/8\">http://www.openwall.com/lists/oss-security/2016/06/23/8</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-87r7-q54j-f9qg\">https://github.com/advisories/GHSA-87r7-q54j-f9qg</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg",
            "title": "[python-muranoclient] OpenStack Murano Code Execution",
            "date_modified": "2023-07-31T23:11:20.000Z",
            "date_published": "2022-05-17T03:48:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Murano before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), Murano-dashboard before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), and python-muranoclient before 0.7.3 (liberty) and 0.8.x before 0.8.5 (mitaka) improperly use loaders inherited from yaml.Loader when parsing MuranoPL and UI files, which allows remote attackers to create arbitrary Python objects and execute arbitrary code via crafted extended YAML tags in UI definitions in packages.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-4972\">https://nvd.nist.gov/vuln/detail/CVE-2016-4972</a></li>\n<li><a href=\"https://bugs.launchpad.net/murano/+bug/1586079\">https://bugs.launchpad.net/murano/+bug/1586079</a></li>\n<li><a href=\"https://bugs.launchpad.net/python-muranoclient/+bug/1586078\">https://bugs.launchpad.net/python-muranoclient/+bug/1586078</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2016/06/23/8\">http://www.openwall.com/lists/oss-security/2016/06/23/8</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-87r7-q54j-f9qg\">https://github.com/advisories/GHSA-87r7-q54j-f9qg</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg",
            "title": "[murano-dashboard] OpenStack Murano Code Execution",
            "date_modified": "2023-07-31T23:11:20.000Z",
            "date_published": "2022-05-17T03:48:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Murano before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), Murano-dashboard before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), and python-muranoclient before 0.7.3 (liberty) and 0.8.x before 0.8.5 (mitaka) improperly use loaders inherited from yaml.Loader when parsing MuranoPL and UI files, which allows remote attackers to create arbitrary Python objects and execute arbitrary code via crafted extended YAML tags in UI definitions in packages.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-4972\">https://nvd.nist.gov/vuln/detail/CVE-2016-4972</a></li>\n<li><a href=\"https://bugs.launchpad.net/murano/+bug/1586079\">https://bugs.launchpad.net/murano/+bug/1586079</a></li>\n<li><a href=\"https://bugs.launchpad.net/python-muranoclient/+bug/1586078\">https://bugs.launchpad.net/python-muranoclient/+bug/1586078</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2016/06/23/8\">http://www.openwall.com/lists/oss-security/2016/06/23/8</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-87r7-q54j-f9qg\">https://github.com/advisories/GHSA-87r7-q54j-f9qg</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg",
            "title": "[murano-dashboard] OpenStack Murano Code Execution",
            "date_modified": "2023-07-31T23:11:20.000Z",
            "date_published": "2022-05-17T03:48:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>OpenStack Murano before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), Murano-dashboard before 1.0.3 (liberty) and 2.x before 2.0.1 (mitaka), and python-muranoclient before 0.7.3 (liberty) and 0.8.x before 0.8.5 (mitaka) improperly use loaders inherited from yaml.Loader when parsing MuranoPL and UI files, which allows remote attackers to create arbitrary Python objects and execute arbitrary code via crafted extended YAML tags in UI definitions in packages.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-4972\">https://nvd.nist.gov/vuln/detail/CVE-2016-4972</a></li>\n<li><a href=\"https://bugs.launchpad.net/murano/+bug/1586079\">https://bugs.launchpad.net/murano/+bug/1586079</a></li>\n<li><a href=\"https://bugs.launchpad.net/python-muranoclient/+bug/1586078\">https://bugs.launchpad.net/python-muranoclient/+bug/1586078</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2016/06/23/8\">http://www.openwall.com/lists/oss-security/2016/06/23/8</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-87r7-q54j-f9qg\">https://github.com/advisories/GHSA-87r7-q54j-f9qg</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg",
            "title": "[murano] OpenStack Murano Code Execution",
            "date_modified": "2023-07-31T23:11:20.000Z",
            "date_published": "2022-05-17T03:48:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-87r7-q54j-f9qg"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.9.x before 1.9.2, when ModelAdmin.save_as is set to True, allows remote authenticated users to bypass intended access restrictions and create ModelAdmin objects via the \"Save as New\" option when editing objects and leveraging the \"change\" permission.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-2048\">https://nvd.nist.gov/vuln/detail/CVE-2016-2048</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/feb/01/releases-192-and-189/\">https://www.djangoproject.com/weblog/2016/feb/01/releases-192-and-189/</a></li>\n<li><a href=\"https://web.archive.org/web/20210123075529/http://www.securityfocus.com/bid/82329\">https://web.archive.org/web/20210123075529/http://www.securityfocus.com/bid/82329</a></li>\n<li><a href=\"https://web.archive.org/web/20211204051406/http://www.securitytracker.com/id/1034894\">https://web.archive.org/web/20211204051406/http://www.securitytracker.com/id/1034894</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-46x4-9jmv-jc8p\">https://github.com/advisories/GHSA-46x4-9jmv-jc8p</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-46x4-9jmv-jc8p",
            "title": "[django] Django Access Restrictions Bypass ",
            "date_modified": "2023-07-31T22:43:11.000Z",
            "date_published": "2022-05-17T03:43:00.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-46x4-9jmv-jc8p"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django before 1.8.x before 1.8.16, 1.9.x before 1.9.11, and 1.10.x before 1.10.3, when settings.DEBUG is True, allow remote attackers to conduct DNS rebinding attacks by leveraging failure to validate the HTTP Host header against settings.ALLOWED_HOSTS.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-9014\">https://nvd.nist.gov/vuln/detail/CVE-2016-9014</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/nov/01/security-releases/\">https://www.djangoproject.com/weblog/2016/nov/01/security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3835\">http://www.debian.org/security/2017/dsa-3835</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-3115-1\">http://www.ubuntu.com/usn/USN-3115-1</a></li>\n<li><a href=\"https://web.archive.org/web/20210123185619/http://www.securityfocus.com/bid/94068\">https://web.archive.org/web/20210123185619/http://www.securityfocus.com/bid/94068</a></li>\n<li><a href=\"https://web.archive.org/web/20211204043252/http://www.securitytracker.com/id/1037159\">https://web.archive.org/web/20211204043252/http://www.securitytracker.com/id/1037159</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-3f2c-jm6v-cr35\">https://github.com/advisories/GHSA-3f2c-jm6v-cr35</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-3f2c-jm6v-cr35",
            "title": "[django] Django DNS Rebinding Vulnerability",
            "date_modified": "2023-07-31T22:41:13.000Z",
            "date_published": "2022-05-17T00:27:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-3f2c-jm6v-cr35"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django before 1.8.x before 1.8.16, 1.9.x before 1.9.11, and 1.10.x before 1.10.3, when settings.DEBUG is True, allow remote attackers to conduct DNS rebinding attacks by leveraging failure to validate the HTTP Host header against settings.ALLOWED_HOSTS.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-9014\">https://nvd.nist.gov/vuln/detail/CVE-2016-9014</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/nov/01/security-releases/\">https://www.djangoproject.com/weblog/2016/nov/01/security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3835\">http://www.debian.org/security/2017/dsa-3835</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-3115-1\">http://www.ubuntu.com/usn/USN-3115-1</a></li>\n<li><a href=\"https://web.archive.org/web/20210123185619/http://www.securityfocus.com/bid/94068\">https://web.archive.org/web/20210123185619/http://www.securityfocus.com/bid/94068</a></li>\n<li><a href=\"https://web.archive.org/web/20211204043252/http://www.securitytracker.com/id/1037159\">https://web.archive.org/web/20211204043252/http://www.securitytracker.com/id/1037159</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-3f2c-jm6v-cr35\">https://github.com/advisories/GHSA-3f2c-jm6v-cr35</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-3f2c-jm6v-cr35",
            "title": "[django] Django DNS Rebinding Vulnerability",
            "date_modified": "2023-07-31T22:41:13.000Z",
            "date_published": "2022-05-17T00:27:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-3f2c-jm6v-cr35"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django before 1.8.x before 1.8.16, 1.9.x before 1.9.11, and 1.10.x before 1.10.3, when settings.DEBUG is True, allow remote attackers to conduct DNS rebinding attacks by leveraging failure to validate the HTTP Host header against settings.ALLOWED_HOSTS.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-9014\">https://nvd.nist.gov/vuln/detail/CVE-2016-9014</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OG5ROMUPS6C7BXELD3TAUUH7OBYV56WQ/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QXDKJYHN74BWY3P7AR2UZDVJREQMRE6S/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/nov/01/security-releases/\">https://www.djangoproject.com/weblog/2016/nov/01/security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2017/dsa-3835\">http://www.debian.org/security/2017/dsa-3835</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-3115-1\">http://www.ubuntu.com/usn/USN-3115-1</a></li>\n<li><a href=\"https://web.archive.org/web/20210123185619/http://www.securityfocus.com/bid/94068\">https://web.archive.org/web/20210123185619/http://www.securityfocus.com/bid/94068</a></li>\n<li><a href=\"https://web.archive.org/web/20211204043252/http://www.securitytracker.com/id/1037159\">https://web.archive.org/web/20211204043252/http://www.securitytracker.com/id/1037159</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-3f2c-jm6v-cr35\">https://github.com/advisories/GHSA-3f2c-jm6v-cr35</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-3f2c-jm6v-cr35",
            "title": "[django] Django DNS Rebinding Vulnerability",
            "date_modified": "2023-07-31T22:41:13.000Z",
            "date_published": "2022-05-17T00:27:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-3f2c-jm6v-cr35"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The utils.http.is_safe_url function in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to redirect users to arbitrary web sites and conduct phishing attacks or possibly conduct cross-site scripting (XSS) attacks via a URL containing basic authentication, as demonstrated by <a href=\"http://mysite.example.com%5C@attacker.com\">http://mysite.example.com\\@attacker.com</a>.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-2512\">https://nvd.nist.gov/vuln/detail/CVE-2016-2512</a></li>\n<li><a href=\"https://github.com/django/django/commit/c5544d289233f501917e25970c03ed444abbd4f0\">https://github.com/django/django/commit/c5544d289233f501917e25970c03ed444abbd4f0</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/mar/01/security-releases/\">https://www.djangoproject.com/weblog/2016/mar/01/security-releases/</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0502.html\">http://rhn.redhat.com/errata/RHSA-2016-0502.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0504.html\">http://rhn.redhat.com/errata/RHSA-2016-0504.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0505.html\">http://rhn.redhat.com/errata/RHSA-2016-0505.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0506.html\">http://rhn.redhat.com/errata/RHSA-2016-0506.html</a></li>\n<li><a href=\"http://www.debian.org/security/2016/dsa-3544\">http://www.debian.org/security/2016/dsa-3544</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html\">http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-1\">http://www.ubuntu.com/usn/USN-2915-1</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-2\">http://www.ubuntu.com/usn/USN-2915-2</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-3\">http://www.ubuntu.com/usn/USN-2915-3</a></li>\n<li><a href=\"https://web.archive.org/web/20210123090815/http://www.securityfocus.com/bid/83879\">https://web.archive.org/web/20210123090815/http://www.securityfocus.com/bid/83879</a></li>\n<li><a href=\"https://web.archive.org/web/20210413200202/http://www.securitytracker.com/id/1035152\">https://web.archive.org/web/20210413200202/http://www.securitytracker.com/id/1035152</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-pw27-w7w4-9qc7\">https://github.com/advisories/GHSA-pw27-w7w4-9qc7</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-pw27-w7w4-9qc7",
            "title": "[django] Django XSS Vulnerability",
            "date_modified": "2023-07-31T22:18:52.000Z",
            "date_published": "2022-05-17T01:09:58.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-pw27-w7w4-9qc7"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The utils.http.is_safe_url function in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to redirect users to arbitrary web sites and conduct phishing attacks or possibly conduct cross-site scripting (XSS) attacks via a URL containing basic authentication, as demonstrated by <a href=\"http://mysite.example.com%5C@attacker.com\">http://mysite.example.com\\@attacker.com</a>.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-2512\">https://nvd.nist.gov/vuln/detail/CVE-2016-2512</a></li>\n<li><a href=\"https://github.com/django/django/commit/c5544d289233f501917e25970c03ed444abbd4f0\">https://github.com/django/django/commit/c5544d289233f501917e25970c03ed444abbd4f0</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/mar/01/security-releases/\">https://www.djangoproject.com/weblog/2016/mar/01/security-releases/</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0502.html\">http://rhn.redhat.com/errata/RHSA-2016-0502.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0504.html\">http://rhn.redhat.com/errata/RHSA-2016-0504.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0505.html\">http://rhn.redhat.com/errata/RHSA-2016-0505.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0506.html\">http://rhn.redhat.com/errata/RHSA-2016-0506.html</a></li>\n<li><a href=\"http://www.debian.org/security/2016/dsa-3544\">http://www.debian.org/security/2016/dsa-3544</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html\">http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-1\">http://www.ubuntu.com/usn/USN-2915-1</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-2\">http://www.ubuntu.com/usn/USN-2915-2</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-3\">http://www.ubuntu.com/usn/USN-2915-3</a></li>\n<li><a href=\"https://web.archive.org/web/20210123090815/http://www.securityfocus.com/bid/83879\">https://web.archive.org/web/20210123090815/http://www.securityfocus.com/bid/83879</a></li>\n<li><a href=\"https://web.archive.org/web/20210413200202/http://www.securitytracker.com/id/1035152\">https://web.archive.org/web/20210413200202/http://www.securitytracker.com/id/1035152</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-pw27-w7w4-9qc7\">https://github.com/advisories/GHSA-pw27-w7w4-9qc7</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-pw27-w7w4-9qc7",
            "title": "[django] Django XSS Vulnerability",
            "date_modified": "2023-07-31T22:18:52.000Z",
            "date_published": "2022-05-17T01:09:58.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-pw27-w7w4-9qc7"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The password hasher in <code>contrib/auth/hashers.py</code> in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to enumerate users via a timing attack involving login requests.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-2513\">https://nvd.nist.gov/vuln/detail/CVE-2016-2513</a></li>\n<li><a href=\"https://github.com/django/django/commit/67b46ba7016da2d259c1ecc7d666d11f5e1cfaab\">https://github.com/django/django/commit/67b46ba7016da2d259c1ecc7d666d11f5e1cfaab</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/mar/01/security-releases/\">https://www.djangoproject.com/weblog/2016/mar/01/security-releases/</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0502.html\">http://rhn.redhat.com/errata/RHSA-2016-0502.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0504.html\">http://rhn.redhat.com/errata/RHSA-2016-0504.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0505.html\">http://rhn.redhat.com/errata/RHSA-2016-0505.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0506.html\">http://rhn.redhat.com/errata/RHSA-2016-0506.html</a></li>\n<li><a href=\"http://www.debian.org/security/2016/dsa-3544\">http://www.debian.org/security/2016/dsa-3544</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html\">http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/83878\">http://www.securityfocus.com/bid/83878</a></li>\n<li><a href=\"http://www.securitytracker.com/id/1035152\">http://www.securitytracker.com/id/1035152</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-1\">http://www.ubuntu.com/usn/USN-2915-1</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-2\">http://www.ubuntu.com/usn/USN-2915-2</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-3\">http://www.ubuntu.com/usn/USN-2915-3</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fp6p-5xvw-m74f\">https://github.com/advisories/GHSA-fp6p-5xvw-m74f</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fp6p-5xvw-m74f",
            "title": "[django] Django User Enumeration Vulnerability",
            "date_modified": "2023-07-31T22:16:33.000Z",
            "date_published": "2022-05-17T01:09:58.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fp6p-5xvw-m74f"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The password hasher in <code>contrib/auth/hashers.py</code> in Django before 1.8.10 and 1.9.x before 1.9.3 allows remote attackers to enumerate users via a timing attack involving login requests.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-2513\">https://nvd.nist.gov/vuln/detail/CVE-2016-2513</a></li>\n<li><a href=\"https://github.com/django/django/commit/67b46ba7016da2d259c1ecc7d666d11f5e1cfaab\">https://github.com/django/django/commit/67b46ba7016da2d259c1ecc7d666d11f5e1cfaab</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2016/mar/01/security-releases/\">https://www.djangoproject.com/weblog/2016/mar/01/security-releases/</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0502.html\">http://rhn.redhat.com/errata/RHSA-2016-0502.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0504.html\">http://rhn.redhat.com/errata/RHSA-2016-0504.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0505.html\">http://rhn.redhat.com/errata/RHSA-2016-0505.html</a></li>\n<li><a href=\"http://rhn.redhat.com/errata/RHSA-2016-0506.html\">http://rhn.redhat.com/errata/RHSA-2016-0506.html</a></li>\n<li><a href=\"http://www.debian.org/security/2016/dsa-3544\">http://www.debian.org/security/2016/dsa-3544</a></li>\n<li><a href=\"http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html\">http://www.oracle.com/technetwork/topics/security/bulletinapr2016-2952098.html</a></li>\n<li><a href=\"http://www.securityfocus.com/bid/83878\">http://www.securityfocus.com/bid/83878</a></li>\n<li><a href=\"http://www.securitytracker.com/id/1035152\">http://www.securitytracker.com/id/1035152</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-1\">http://www.ubuntu.com/usn/USN-2915-1</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-2\">http://www.ubuntu.com/usn/USN-2915-2</a></li>\n<li><a href=\"http://www.ubuntu.com/usn/USN-2915-3\">http://www.ubuntu.com/usn/USN-2915-3</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fp6p-5xvw-m74f\">https://github.com/advisories/GHSA-fp6p-5xvw-m74f</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fp6p-5xvw-m74f",
            "title": "[django] Django User Enumeration Vulnerability",
            "date_modified": "2023-07-31T22:16:33.000Z",
            "date_published": "2022-05-17T01:09:58.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fp6p-5xvw-m74f"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<h2 id=\"impact\">Impact</h2>\n<p>If configured to send emails using TLS, Sydent does not verify SMTP servers' certificates. This makes  Sydent's emails vulnerable to interception via a <a href=\"https://en.wikipedia.org/wiki/Man-in-the-middle_attack\">man-in-the-middle (MITM) attack</a>. Attackers with privileged access to the network can intercept room invitations and address confirmation emails.</p>\n<p>CVSS 3.1 overall score: 3.3 - <a href=\"https://nvd.nist.gov/vuln-metrics/cvss/v3-calculator?vector=AV:A/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:N/CR:L/IR:L/AR:X/MAV:A/MAC:H/MPR:N/MUI:N/MS:C/MC:L/MI:L/MA:N&amp;version=3.1\">AV:A/AC:L/PR:N/UI:N/S:C/C:H/I:H/A:N/CR:L/IR:L/AR:X/MAV:A/MAC:H/MPR:N/MUI:N/MS:C/MC:L/MI:L/MA:N</a></p>\n<p><em>Reported by Martin Schobert, <a href=\"https://pentagrid.ch/\">Pentagrid AG</a>.</em></p>\n<h3 id=\"details\">Details</h3>\n<p>Sydent can be configured to send emails over a TLS-encrypted socket by setting</p>\n<pre><code class=\"language-yaml\">email:\n    tlsmode: \"TLS\"  # or the legacy value \"SSL\"\n</code></pre>\n<p>in its config file. Alternatively it can be configured to use <a href=\"https://en.wikipedia.org/wiki/Opportunistic_TLS\">Opportunistic TLS</a> by setting</p>\n<pre><code class=\"language-yaml\">email:\n    tlsmode: \"STARTTLS\"\n</code></pre>\n<p>In both situations, Sydent will encrypt its communication with the SMTP server when sending emails. <strong>In affected versions, Sydent will not verify the destination server's certificate.</strong></p>\n<h3 id=\"vulnerability\">Vulnerability</h3>\n<p>Sydent sends email for two purposes:</p>\n<ul>\n<li>to inform a third party that they have been invited to a Matrix room by their email address; and</li>\n<li>to verify that a given Matrix user controls an email address.</li>\n</ul>\n<p>Therefore, attackers capable of running a MITM attack can</p>\n<ol>\n<li>Intercept room invitations sent to an email address. The invitation includes <ul>\n<li>the room ID and its avatar, and</li>\n<li>the inviter's username, displayname and their avatar, and</li>\n<li>credentials for a guest Matrix account on the inviter's homeserver.</li>\n</ul>\n</li>\n<li>Intercept address ownership confirmation emails. This would allow the attacker to falsely claim ownership of the indented recipient's Matrix account, if that account was permitted to log in using an email address and no other authentication factors.</li>\n</ol>\n<h3 id=\"patches\">Patches</h3>\n<p>This is patched in <a href=\"https://github.com/matrix-org/sydent/releases/tag/v2.5.6\">Sydent 2.5.6</a>, see PR <a href=\"https://github.com/matrix-org/sydent/pull/574\">https://github.com/matrix-org/sydent/pull/574</a>.</p>\n<p>When patching, make sure that Sydent trusts the certificate of the server it is connecting to. This should happen automatically when using properly issued certificates. If you are using self-signed certificates, make sure to copy your Certification Authority certificate, or your self signed certificate if using only one, to the trust store of your operating system.</p>\n<h3 id=\"workarounds\">Workarounds</h3>\n<p>One can ensure Sydent's emails fail to send by setting the configured SMTP server to a loopback or <a href=\"https://datatracker.ietf.org/doc/html/rfc1918#section-3\">non-routable</a> address under your control which does not have a listening SMTP server. For example:</p>\n<pre><code class=\"language-yaml\">email:\n    smtphost: \"localhost\"  # Assuming there is no SMTP server listening on localhost\n</code></pre>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://github.com/matrix-org/sydent/pull/574\">https://github.com/matrix-org/sydent/pull/574</a> implements the fix.</li>\n<li><a href=\"https://github.com/matrix-org/sydent/releases/tag/v2.5.6\">https://github.com/matrix-org/sydent/releases/tag/v2.5.6</a> is the release including this fix.</li>\n<li><a href=\"https://docs.python.org/3/library/ssl.html?highlight=ssl#security-considerations\">https://docs.python.org/3/library/ssl.html?highlight=ssl#security-considerations</a> details the best-practice advice on how to use the standard library <code>smtp</code> module safely.</li>\n<li><a href=\"https://peps.python.org/pep-0476/\">https://peps.python.org/pep-0476/</a> (accepted) proposed enabling TLS certificate verification by default  in standard library HTTP clients.</li>\n<li><a href=\"https://github.com/python/cpython/issues/91826\">https://github.com/python/cpython/issues/91826</a> discusses enabling TLS certificate verification by default in the Python standard library, for SMTP and other protocols.</li>\n</ul>\n<h2 id=\"for-more-information\">For more information</h2>\n<p>If you have any questions or comments about this advisory, e-mail us at <a href=\"mailto:security@matrix.org\">security@matrix.org</a>.</p>\n<h3 id=\"references-1\">References</h3>\n<ul>\n<li><a href=\"https://github.com/matrix-org/sydent/security/advisories/GHSA-p6hw-wm59-3g5g\">https://github.com/matrix-org/sydent/security/advisories/GHSA-p6hw-wm59-3g5g</a></li>\n<li><a href=\"https://github.com/python/cpython/issues/91826\">https://github.com/python/cpython/issues/91826</a></li>\n<li><a href=\"https://github.com/matrix-org/sydent/pull/574\">https://github.com/matrix-org/sydent/pull/574</a></li>\n<li><a href=\"https://github.com/matrix-org/sydent/commit/1cd748307c6b168b66154e6c4db715d4b9551261\">https://github.com/matrix-org/sydent/commit/1cd748307c6b168b66154e6c4db715d4b9551261</a></li>\n<li><a href=\"https://docs.python.org/3/library/ssl.html?highlight=ssl#security-considerations\">https://docs.python.org/3/library/ssl.html?highlight=ssl#security-considerations</a></li>\n<li><a href=\"https://github.com/matrix-org/sydent/releases/tag/v2.5.6\">https://github.com/matrix-org/sydent/releases/tag/v2.5.6</a></li>\n<li><a href=\"https://peps.python.org/pep-0476/\">https://peps.python.org/pep-0476/</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-p6hw-wm59-3g5g\">https://github.com/advisories/GHSA-p6hw-wm59-3g5g</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-p6hw-wm59-3g5g",
            "title": "[matrix-sydent] Sydent does not verify email server certificates",
            "date_modified": "2023-07-31T22:03:45.000Z",
            "date_published": "2023-07-31T22:03:44.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-p6hw-wm59-3g5g"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Chameleon (five.pt) in Plone 5.0rc1 through 5.1a1 allows remote authenticated users to bypass Restricted Python by leveraging permissions to create or edit templates.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-4043\">https://nvd.nist.gov/vuln/detail/CVE-2016-4043</a></li>\n<li><a href=\"https://plone.org/security/hotfix/20160419/bypass-restricted-python\">https://plone.org/security/hotfix/20160419/bypass-restricted-python</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2016/04/20/3\">http://www.openwall.com/lists/oss-security/2016/04/20/3</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-6h8x-73fx-q2h9\">https://github.com/advisories/GHSA-6h8x-73fx-q2h9</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-6h8x-73fx-q2h9",
            "title": "[plone] Chameleon in Plone allows Authentication Bypass",
            "date_modified": "2023-07-31T21:08:40.000Z",
            "date_published": "2022-05-17T02:57:32.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-6h8x-73fx-q2h9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>python-jose before 1.3.2 allows attackers to have unspecified impact by leveraging failure to use a constant time comparison for HMAC keys.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2016-7036\">https://nvd.nist.gov/vuln/detail/CVE-2016-7036</a></li>\n<li><a href=\"https://github.com/mpdavis/python-jose/pull/35/commits/89b46353b9f611e9da38de3d2fedf52331167b93\">https://github.com/mpdavis/python-jose/pull/35/commits/89b46353b9f611e9da38de3d2fedf52331167b93</a></li>\n<li><a href=\"https://github.com/mpdavis/python-jose/releases/tag/1.3.2\">https://github.com/mpdavis/python-jose/releases/tag/1.3.2</a></li>\n<li><a href=\"https://web.archive.org/web/20210123221523/http://www.securityfocus.com/bid/95845\">https://web.archive.org/web/20210123221523/http://www.securityfocus.com/bid/95845</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-w799-prg3-cx77\">https://github.com/advisories/GHSA-w799-prg3-cx77</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-w799-prg3-cx77",
            "title": "[python-jose] python-jose failure to use a constant time comparison for HMAC keys",
            "date_modified": "2023-07-31T21:03:31.000Z",
            "date_published": "2022-05-17T03:02:29.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-w799-prg3-cx77"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}